src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -159,7 +160,7 @@ class ChartBuilder:  # pylint: disable = too-many-instance-attributes
     release_name: str
     config_defaults: DeploymentDefaults
     deploy_set: Optional[DeploySet]
-    namespace: str
+    namespace: Optional[str]
```

Comment:
Because that's what `get_namespace` returns, it also doesn't seem to be mandatory in the schema (both in our project.yml's and for `namespaceSelector`)

Reply:
I think we need to keep the logic for determining the namespace central. This is the only other place in the code: https://github.com/Vandebron/mpyl/blob/cc158eecaa2f76139814e5bc9288580b1a33513c/src/mpyl/steps/deploy/k8s/__init__.py#L79
I suppose you also want to fall back on `project.name` in the absence of an explicitly defined namespace?
You probably don't want `None` as namespace here either https://github.com/Vandebron/mpyl/blob/f54c51b5e624785a34f2623d714f2d313028b583/src/mpyl/steps/deploy/k8s/resources/prometheus.py#L60